packages/discord.js/src/structures/ClientApplication.js

```diff
@@ -236,6 +236,36 @@ class ClientApplication extends Application {
       this.roleConnectionsVerificationURL ??= null;
     }
 
+    if ('event_webhooks_url' in data) {
+      /**
+       * This application's event webhooks URL
+       * @type {?string}
+       */
+      this.eventWebhooksURL = data.event_webhooks_url;
+    } else {
+      this.eventWebhooksURL ??= null;
+    }
+
+    if ('event_webhooks_status' in data) {
+      /**
+       * This application's event webhooks status
+       * @type {?ApplicationEventWebhookStatus}
+       */
+      this.eventWebhooksStatus = data.event_webhooks_status;
+    } else {
+      this.eventWebhooksStatus ??= null;
```

Comment:
Is it supposed to be nullable? The documentation says it'll always be present

Reply:
`name`, `icon`, `description`, `bot_public`, etc... are all documented to be always be present, but our docs (and the code) state them to be nullable. So I tried to stick with that. Though I can't imagine when they can actually be null? Most likely in case of the structure being a partial?

Reply:
The READY gateway event sends a partial application object only. That’s why most of the properties are nullable and need a fetch of the application to exist first.